optimum/exporters/openvino/model_configs.py

```diff
@@ -3001,3 +3002,16 @@ def patch_model_for_export(
         self, model: Union["PreTrainedModel", "TFPreTrainedModel"], model_kwargs: Optional[Dict[str, Any]] = None
     ) -> "ModelPatcher":
         return DeepseekPatcher(self, model, model_kwargs=model_kwargs)
+
+
+@register_in_tasks_manager("got-ocr2", *["image-to-text", "image-text-to-text"], library_name="transformers")
+class GotOCR2OpenVINOConfig(LlavaOpenVINOConfig):
+    MIN_TRANSFORMERS_VERSION = "4.49.0"
+
+    def patch_model_for_export(
+        self, model: Union["PreTrainedModel", "TFPreTrainedModel"], model_kwargs: Optional[Dict[str, Any]] = None
+    ):
+        model_kwargs = model_kwargs or {}
+        if self._behavior != LlavaConfigBehavior.VISION_EMBEDDINGS:
```

Comment:
why is `VISION_EMBEDDINGS` not added directly to `ConfigBehavior` ? using `LlavaConfigBehavior` looks wrong here

Reply:
not sure that I understand your comment, there is no separated config behaviour for this model class. it inherits it from parent class - LllavaOpenVINOConfig. vision embeddings is part of it and its LlavaConfigBehavior. This overriding required to use got-ocr specific patching instead of llava (because they are different)

Reply:
every multimodal model may have own partition. it is difficult to maintain everything using only one enum class and will required additional logic for maintenance and validation of model specific structure. 

it maybe refactored later if you have better solution